app/views/episode_metrics/_main_card.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= turbo_frame_tag "main_card" do %>
```

Comment:
follows the pattern of `podcast_metrics/_main_card`. i am able to specify which tabs and cards/routes are used.